### PR TITLE
Peer session id component

### PR DESCRIPTION
**Pull request by Ibhassan01**
Screenshot (78)

### CODE REVIEW

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Please remove the hard codes and make it usable by third party using props

**Review by annonymousauthority — APPROVED** [review]
LG

File: development/src/components/peerId_comp.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import React, { useState } from 'react';
+import Image from 'next/image';
+import { generatePeerIdCharacter } from '../../src/composables/peerIdGenerator';
+
+import closeIconBlack from '../../public/assets/onboardingIcons/close_black.png';
+import closeIconWhite from '../../public/assets/onboardingIcons/close_light.png';
+
+function PeerId() {
+    const [isOpen, setIsOpen] = useState(true);
+    const [peerSessionId, setPeerSessionId] = useState(generatePeerIdCharacter());
```
Comment (7malikk) [inline development/src/components/peerId_comp.js:11]: setState is asynchronous, this is error prone

try utilizing a useEffect that sets the state to the generatedPeerID on render

File: development/src/components/peerId_comp.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    if (!isOpen) {
+        return null;
+    }
+
+    return (
+        <div className="bg-white text-[#0E0C46] dark:bg-[#504F5F]  dark:text-white p-4 flex flex-col rounded-lg">
+            <div className="flex w-full justify-between">
+                <div className="font-nohemi font-bold leading-8">Peer Session Created</div>
+                <button onClick={handleClose}><Image src={closeIconBlack} className="w-5 h-5" /></button>
```
Comment (7malikk) [inline development/src/components/peerId_comp.js:38]: implement the dark theme mode for the icon
refer to other components for an idea

**Review by 7malikk — APPROVED** [review]

**Comment by 7malikk** [comment]
LG

**Pull request merged**